lib/ansible/runner.rb

```diff
@@ -216,6 +216,7 @@ def run_via_cli(hosts, credentials, env_vars, extra_vars, tags: nil, ansible_run
         params = runner_params(base_dir, ansible_runner_method, playbook_or_role_args, verbosity)
 
         begin
+          fetch_galaxy_roles(playbook_or_role_args)
```

Comment:
@carbonin This may change the nature of provider-based playbooks where they were expecting a one-time galaxy install at build time, and now it's that plus all the time.